docs/src/modules/pageDemo.js

```diff
@@ -591,98 +591,122 @@ class PageDemo extends React.PureComponent {
+                                                accessor: 'birthday',
+                                                fieldName: 'Birthday',
+                                                header: 'Training',
+                                                iconType: 'chair',
+                                                iconColor: 'alert',
+                                                iconSize: 22,
                                             }, {
-                                                campus: 'Lake Forest',
-                                                createdOn: 1159668810,
-                                                id: 2,
-                                                name: 'Second Time Visitor',
+                                                accessor: 'homeCampus',
+                                                fieldName: 'Home Campus',
+                                                header: 'Campus',
+                                                iconType: 'church',
+                                                iconSize: 22,
+                                            },
+                                        ],
+                                    }, {
+                                        header: 'Column 4',
+                                        rows: [
+                                            {
+                                                accessor: 'birthday',
+                                                fieldName: 'Birthday',
+                                                header: 'Training',
+                                                iconType: 'chair',
+                                                iconColor: 'alert',
+                                                iconSize: 22,
                                             }, {
-                                                campus: 'Anaheim',
-                                                createdOn: 1152668810,
-                                                id: 3,
-                                                name: 'Class 101 Invite',
+                                                accessor: 'homeCampus',
+                                                fieldName: 'Home Campus',
+                                                header: 'Campus',
+                                                iconType: 'church',
+                                                iconSize: 22,
                                             },
-                                        ]}
-                                    />
-                                )}
-                            <Page.DataGroups
-                                style={{
-                                    margin: '22px 0'
+                                        ],
+                                    },
+                                ]}
+                                data={{
+                                    birthday: '23/01/1990',
```

Comment:
Can we do `1990-01-23` for a common format intelligible and agreeable to devs all over the world? =)

Reply:
changed to this and also added the accessor function statement